appserver/tests/tck/embedded_ejb_smoke/ejb_lite_basic/src/main/java/com/sun/ts/lib/harness/ServiceEETest.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2007, 2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Requires copyright year update

Reply:
I just copied these over without changes, so I think it doesn't need updating of copyright year. I can do it, of course, but I'm not sure it's really needed